src/WixToolset.Core.WindowsInstaller/Bind/CreateOutputFromIRCommand.cs

```
@@ -0,0 +1,230 @@
+                    }
+                    else
+                    {
+                        sequenceTableDefinition = this.TableDefinitions["InstallExecuteSequence"];
+                    }
+                    break;
+                case SequenceTable.InstallUISequence:
+                    if (OutputType.Module == output.Type)
+                    {
+                        output.EnsureTable(this.TableDefinitions["InstallUISequence"]);
+                        sequenceTableDefinition = this.TableDefinitions["ModuleInstallUISequence"];
+                    }
+                    else
+                    {
+                        sequenceTableDefinition = this.TableDefinitions["InstallUISequence"];
+                    }
+                    break;
+            }
+
+            // create the action sequence row in the output
+            var sequenceTable = output.EnsureTable(sequenceTableDefinition);
+            var row = sequenceTable.CreateRow(actionRow.SourceLineNumbers);
+
+            if (SectionType.Module == this.Section.Type)
+            {
+                row[0] = actionRow.Action;
+                if (0 != actionRow.Sequence)
+                {
+                    row[1] = actionRow.Sequence;
+                }
+                else
+                {
+                    bool after = (null == actionRow.Before);
+                    row[2] = after ? actionRow.After : actionRow.Before;
+                    row[3] = after ? 1 : 0;
+                }
+                row[4] = actionRow.Condition;
+            }
+            else
+            {
+                row[0] = actionRow.Action;
+                row[1] = actionRow.Condition;
+                row[2] = actionRow.Sequence;
+            }
+        }
+
+        private void AddTupleDefaultly(IntermediateTuple tuple, Output output)
+        {
+            if (!this.TableDefinitions.TryGet(tuple.Definition.Name, out var tableDefinition))
```

It feels wrong to silently ignore the tuple if we couldn't find the table definition.

---

Fixed in https://github.com/wixtoolset/Core/pull/121